plugins/file/file_output.go

```diff
@@ -257,11 +296,15 @@ func (o *FileOutput) committer(or OutputRunner, wg *sync.WaitGroup) {
 				// Channel is closed => we're shutting down, exit cleanly.
 				break
 			}
+			if err = o.tryRotateFile(); err != nil {
+				panic(fmt.Sprintf("FileOutput unable to rotate file '%s': %s",
```

Comment:
But, I initially followed the same direction as the SIGHUP handling code:
https://github.com/bbinet/heka/blob/fileoutput-rotation/plugins/file/file_output.go#L318

So, do you confirm I should change this to LogError?